### ISSUE TITLE
Add ability to name & log an arbitrary number of temperature probes

### ISSUE THREAD

**Issue by speshak** [thorrak/fermentrack#492]
I have additional temp probes that aren't one of the beer/fridge/room options. I'm using fermentrack/brewpi-8266 to control glycol chill loops on my fermenters.  I'm ok with the control probes not have "correct" names (for example, my fermenter probe is configured as the fridge probe in fermentrack), but I'd like to at least be able to log the data from these extra probes.  

I personally have a probe for the coolant bath and for the post-jacket coolant temp, but I think any solution should allow arbitrary names to be set on these non-control probes.

**Comment by uSlackr** [comment]
(sorry if off topic) Why don't the three probes have the right names?  That is configurable

\\Greg

**Comment by speshak** [comment]
I couldn't get it to actually cool when using just a beer temp probe. The interface says that chamber (aka fridge) is required.  Because of that the probe that is in my fermentor is my chamber probe. (I don't actually have a chamber, because I'm using glycol)

**Comment by uSlackr** [comment]
I would think the beer probe is the beer probe, the glycol is the chamber/fridge and the outside is the room.  No?

\\Greg

**Comment by speshak** [comment]
Not quite.  The beer is the beer, but the glycol is not the chamber because the glycol is kept at a constant temp by a controller built into the chiller.  The control point is turning the coolant pump on and off to change the fermenter temp. (Said another way: I'm _not_ controlling the temp of the glycol.  I'm using cold glycol to control the temp of the fermenter.)

**Comment by uSlackr** [comment]
Sure. But the glycol is equivalent to the cool fridge air. The coming cycle would just turn on the pump to initiate cooling instead of the fridge.

\\\\ Greg
________________________________

**Comment by speshak** [comment]
I think I'm tracking what you mean now.  I thought there was some reason why I couldn't get it to work that way.  I'll play with it a bit more and see if I can capture the details.  We're straying a bit far afield of the original intent of this issue. 😁

**Comment by thorrak** [comment]
The BrewPi control model directly controls the "fridge" temperature in order to indirectly control the "beer" temperature. This is why the "fridge" sensor is mandatory for a BrewPi controller to work. For most glycol setups, this means that:

- Your "fridge sensor" should actually go in your beer
- There is no "beer sensor" (though you _could_ also have it in your beer)
- The controller handling your fermentation temps _will not_ control the temperature of your actual glycol 

...and the room sensor is (as always) optional.

**Comment by thorrak** [comment]
> We're straying a bit far afield of the original intent of this issue. 😁

How are your additional temperature probes connected? The BrewPi-ESP8266 port only supports reporting the three typical probe types (fridge/beer/room) - are you using additional hardware?

**Comment by speshak** [comment]
The additional probes are just more probes on the OneWire bus.  I am using a custom PCB based off yours that has additional probe connectors.

The probes show up fine as uninitiated devices, I just want to get their data into the CSV.
